api/imp_links.json

```
@@ -1,32 +1,246 @@
+      "url": "https://github.com/linuxfoundation"
+    },
+    {
+      "title": "Linux Foundation blog",
+      "url": "https://linuxfoundation.org/blog/"
+    },
+    {
+      "title": "Linux Foundation Twitter page",
+      "url": "https://twitter.com/linuxfoundation"
+    },
+    {
+      "title": "Linux Foundation LinkedIn page",
+      "url": "https://www.linkedin.com/company/linux-foundation"
+    }
+  ],
+  "Mozilla Foundation": [
+    {
+      "title": "Mozilla Foundation official documentation",
+      "url": "https://developer.mozilla.org/"
+    },
+    {
+      "title": "Mozilla Foundation GitHub repository",
+      "url": "https://github.com/mozilla"
+    },
+    {
+      "title": "Mozilla Foundation blog",
+      "url": "https://blog.mozilla.org/"
+    },
+    {
+      "title": "Mozilla Foundation Twitter page",
+      "url": "https://twitter.com/mozilla"
+    },
+    {
+      "title": "Mozilla Foundation LinkedIn page",
+      "url": "https://www.linkedin.com/company/mozilla-foundation"
+    }
+  ],
+  "Wikimedia Foundation": [
+    {
+      "title": "Wikimedia Foundation official documentation",
+      "url": "https://www.mediawiki.org/wiki/MediaWiki"
+    },
+    {
+      "title": "Wikimedia Foundation GitHub repository",
+      "url": "https://github.com/wikimedia"
+    },
+    {
+      "title": "Wikimedia Foundation blog",
+      "url": "https://blog.wikimedia.org/"
+    },
+    {
+      "title": "Wikimedia Foundation Twitter page",
+      "url": "https://twitter.com/wikimedia"
+    },
+    {
+      "title": "Wikimedia Foundation LinkedIn page",
+      "url": "https://www.linkedin.com/company/wikimedia-foundation"
+    }
+  ],
+  "Internet Archive": [
+    {
+      "title": "Internet Archive official documentation",
+      "url": "https://archive.org/about/faqs/"
+    },
+    {
+      "title": "Internet Archive GitHub repository",
+      "url": "https://github.com/archiveorg"
+    },
+    {
+      "title": "Internet Archive blog",
+      "url": "https://blog.archive.org/"
+    },
+    {
+      "title": "Internet Archive Twitter page",
+      "url": "https://twitter.com/internetarchive"
+    },
+    {
+      "title": "Internet Archive LinkedIn page",
+      "url": "https://www.linkedin.com/company/internet-archive"
+    }
+  ],
+  "Electronic Frontier Foundation": [
+    {
+      "title": "Electronic Frontier Foundation official documentation",
+      "url": "https://www.eff.org/legal/cases/"
+    },
+    {
+      "title": "Electronic Frontier Foundation GitHub repository",
+      "url": "https://github.com/eff"
+    },
+    {
+      "title": "Electronic Frontier Foundation blog",
+      "url": "https://www.eff.org/deeplinks/"
+    },
+    {
+      "title": "Electronic Frontier Foundation Twitter page",
+      "url": "https://twitter.com/eff"
+    },
+    {
+      "title": "Electronic Frontier Foundation LinkedIn page",
+      "url": "https://www.linkedin.com/company/electronic-frontier-foundation"
     }
   ]
-}
+}
```

```suggestion
}

```

Please add a new line at the end of the json file.

---

@LordMashh please only resolve conversation after addressing the review.